### DISCUSSION TITLE
Great program! Works on Intel Mac as well!

### DISCUSSION THREAD

**Discussion by tylerbcrawford** [john-stephens/zebra-mac-label-automator#8] [General]
Just got this up and running on an old Zebra LP2844, on an Intel Mac running Sonoma! I had to modify the file to change the directory to /usr/local/bin and it worked right away! Crystal clear, thank you for your work!

**Comment by john-stephens** [comment]
Great! I'll add your model to the list. Appreciate the feedback.
